src/components/Button.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Button from './Button';
+import { ReactComponent as IconComponent } from '../images/play-large.svg';
+
+export default {
+  title: 'Button',
+  component: Button,
+  argTypes: {
+    iconOnly: {
+      defaultValue: false,
+    },
+    children: {
+      defaultValue: 'Button',
+    },
+    size: {
+      options: ['small', 'big'],
+      control: { type: 'radio' },
+    },
+  },
+};
+
+const Template = (args) => <Button {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  onClick: undefined,
+};
+
+export const Small = Template.bind({});
```

Comment:
You don't need separate `Small`, `Big` stories when you're using `size` argType. Either remove the `size` argType, or add "default" to argType the exported variants (preferred).